owncloudData/src/main/java/com/owncloud/android/data/files/db/FileDao.kt

```diff
@@ -552,6 +556,12 @@ interface FileDao {
             WHERE keepInSync = '1'
         """
 
+        private const val SELECT_FILES_WHERE_LAST_USAGE_IS_OLDER_THAN_GIVEN_TIME = """
+            SELECT *
+            FROM ${ProviderMeta.ProviderTableMeta.FILES_TABLE_NAME}
+            WHERE lastUsage < strftime('%s', 'now') * 1000 - (:milliseconds);
```

Comment:
Why using `strftime` when we can use Kotlin expressions? And the final `;`?

Reply:
Also, you should NOT retrieve av. offline files in this query

Reply:
Done